Source/WebKit/UIProcess/API/glib/WebKitProtocolHandler.cpp

```diff
@@ -426,7 +426,9 @@ void WebKitProtocolHandler::handleGPU(WebKitURISchemeRequest* request)
     addTableRow(displayObject, "Depth"_s, String::number(screenDepth(nullptr)));
     addTableRow(displayObject, "Bits per color component"_s, String::number(screenDepthPerComponent(nullptr)));
     addTableRow(displayObject, "Font Scaling DPI"_s, String::number(fontDPI()));
+#if PLATFORM(WPE) && ENABLE(WPE_PLATFORM)
```

Comment:
Shouldn't this include PLATFORM(GTK) as well?

Reply:
Agreed.